site/sass/globals/_functions.scss

```diff
@@ -18,11 +18,13 @@
   @param {boolean} $rem (optional) - Cast the provided px length or unitless number to a rem. When false, casts to em instead.
 */
 @function pxCast($length, $rem: true) {
+  /* stylelint-disable function-no-unknown */
```

Comment:
Does adding these to the `ignoreFunctions` in `.stylelintrc.json` not work?

Reply:
We could totally add them there instead! I opted to ignore within the bodies of the `pxCast` and `generateScaling` functions since I wouldn't expect the offending functions (`if`, `pxCast`, `unitless`, and `unquote`) to be used elsewhere.